README.rst

```diff
@@ -1,77 +1,105 @@
-
-
 Monitoring-Integration
 =======================
 
+Installation Details
+--------------------
+
+The following steps outlines the procedure to install monitoring-intregration component.
+
+**Note :**
 
-Functionalities
-----------------
+  All the commands mentioned below are run as a regular user that has ``sudo``
+  privileges.
 
-- Create new dashboard in Grafana
 
+1) **Conifgure Graphite**
```

Comment:
@rishubhjain Graphite will be installed as part of monitoring integration, so a user cannot configure it if Graphite is not available. 

This step should be after should go after the installation of monitoring integration.